src/gas/determineGasFeeSuggestions.ts

```diff
@@ -69,7 +75,12 @@ export default async function determineGasFeeSuggestions({
 
   if (isEIP1559Compatible) {
     strategies.push(async () => {
-      const estimates = await fetchGasEstimates(fetchGasEstimatesUrl, clientId);
+      let estimates: GasFeeEstimates;
```

Comment:
@danjm Is this what you had in mind? i.e., do we still want to use `eth_feeHistory` as a fallback for the EIP-1559 Metaswap API, and then continue to use `eth_gasPrice` as an ultimate fallback whether EIP-1559 or not?